notebooks/dc2/catalog_generation/matchingLensGalaxies.ipynb

```diff
@@ -0,0 +1,6293 @@
+    "\n",
+    "CosmoDC2 information is in [Korytov et al. 2019](https://arxiv.org/pdf/1907.06530.pdf)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 81,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## As a proxy for elliptical galaxies we cut of fraction of stellar mass that is in bulge\n",
+    "keep_gals = np.where(full_lens_join['stellar_mass_bulge']/full_lens_join['stellar_mass'] > 0.99)[0]\n",
+    "full_lens_join = full_lens_join.iloc[keep_gals].reset_index(drop=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 82,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Shorten some parameter calls\n",
+    "gal_radius = full_lens_join['morphology/spheroidHalfLightRadius'].values\n",
+    "gal_radius_arcsec = full_lens_join['morphology/spheroidHalfLightRadiusArcsec'].values"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 84,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def calc_mu_e(app_mag, radius_arcsec, redshift):\n",
+    "    \n",
+    "    \"\"\"\n",
+    "    Calculate the mu_e parameter using Eq. 7 in Hyde and Bernardi 2009\n",
+    "    \n",
+    "    Parameters\n",
+    "    ----------\n",
+    "    \n",
+    "    app_mag: float\n",
+    "      Apparent observed LSST r-band magnitude\n",
+    "      \n",
+    "    radius_arcsec: float\n",
+    "      Galaxy radius in arcsec. We use the bulge radius since we want to look at large elliptical galaxies.\n",
+    "      \n",
+    "    redshift: float\n",
+    "      Galaxy redshift\n",
+    "      \n",
+    "    Returns\n",
+    "    -------\n",
+    "    \n",
+    "    mu_e: float\n",
+    "      Mu_e parameter for Fundamental Plane\n",
+    "    \"\"\"\n",
+    "    \n",
+    "    return app_mag + 5*np.log10(radius_arcsec) + 2.5*np.log10(2*np.pi) - 10*np.log10(1+redshift)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 85,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def calc_sigma_fp(radius, mu_e):\n",
+    "    \n",
+    "    \"\"\"\n",
+    "    Calculate velocity dispersion using Fundamental Plane relation in Equation 6 of Hyde and Bernardi 2009\n",
+    "    \n",
+    "    Parameters\n",
+    "    ----------\n",
+    "    \n",
+    "    radius: float\n",
+    "      Galaxy radius in kpc.\n",
+    "      \n",
+    "    mu_e: float\n",
+    "      Mu_e parameter calculated using `calc_mu_e` function\n",
+    "      \n",
+    "    Returns\n",
+    "    -------\n",
+    "    \n",
+    "    sigma_fp: float\n",
+    "      Velocity Dispersion in km/s for galaxies\n",
+    "    \"\"\"\n",
+    "\n",
+    "    a = 1.4335 \n",
+    "    b = 0.3150\n",
+    "    c = -8.8979\n",
+    "    \n",
+    "    print(np.log10(radius))\n",
+    "    print(-b*mu_e)\n",
+    "    print(c)\n",
+    "    \n",
+    "    log_sigma_fp = (np.log10(radius) - b*mu_e - c)/a\n",
```

Comment:
Do you think we should take the coefficient errors into account? Hyde & Bernardi 2009 says the random errors are delta_a ~ 0.02 and delta_b ~ 0.01, and systematic errors are delta_a ~ 0.05 and delta_b ~ 0.02.

Reply:
Up to you. Do you want to include them? I don't have any objections.

Reply:
Actually, I think we just need to include the intrinsic errors (0.0578 for r-band orthogonal). Can we do that please?